src/main/java/seedu/address/commons/core/Messages.java

```diff
@@ -7,7 +7,9 @@
 
     public static final String MESSAGE_UNKNOWN_COMMAND = "Unknown command";
     public static final String MESSAGE_INVALID_COMMAND_FORMAT = "Invalid command format! \n%1$s";
+    public static final String MESSAGE_INVALID_INPUT = "Invalid input value!";
     public static final String MESSAGE_INVALID_PERSON_DISPLAYED_INDEX = "The person index provided is invalid";
     public static final String MESSAGE_PERSONS_LISTED_OVERVIEW = "%1$d persons listed!";
+    public static final String MESSAGE_TAG_NOT_FOUND = "Tag is not found.";
```

Comment:
Just a thought: not sure if you want to remove this message because: (i) you can simply treat such cases as the tags having been removed anyway (e.g. for Zhen Yong's `clearhistory` command we will also get a success message even if no history was actually cleared, i.e. `clearhistory` is the first command) and (ii) this message is only used for the tag command, and is not a message that is likely to be shared across different commands (i.e. does not quite belong in `/commons`

Reply:
i) emm what i imagined is the scenario that user wants to quickly find if a tag exists by using the removeTag command, since the user can redo this command, it's actually equivalent to findTag. Moreover, if the tags are sparsely distributed in the list, e.g. only one tag at the end of list which is super down the bottom, you might need multiple tries to delete the tag. therefore, i think reminding the user that certain tags do not exist is necessary.
ii) emm i m thinking of moving it to RemoveTagCommand class as inside the RemoveTagCommand class there is also a success message.

Reply:
(i) Well, since you find that it resembles a `find tag` command, I agree with you. Consequently, since I have already implemented a separate `find tag` command, do you think that that renders this redundant?
But I guess moving inside `RemoveTagCommand` class is fine, so go ahead